### PR TITLE
Manager page

### PR DESCRIPTION
**Pull request by benjamin-firth**
**What's this PR do?**
This PR adds manager page functionality.  It is rendered in the index.js file, and uses the Hotel and Manager class methods to add the necessary info on the page.
**Where should the reviewer start?**
I'd start on line 80 of index.js and then look at the Manager and Hotel class files.

**How should this be manually tested?**
npm test will show you that the methods work, and maybe spies will show that they've been put on the DOM.

Questions:
@Garrett-Iannuzzi let me know what you think, or if any changes might make sense!

@brittanystoroz @khalidwilliams

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -7,26 +7,57 @@
...
 
-nav {
-  background-color: #5271FF;
-  grid-area: nav;
+button {
+  align-self: flex-end;
+  border-radius: 10%;
+  padding: 5px 15px;
+  margin: 10px;
+}
```
Comment (Garrett-Iannuzzi) [inline src/css/base.scss]: If you'd like you can also use em's here to help with accessibility later on.

File: src/css/base.scss [hunk 2]
```diff
@@ -7,26 +7,57 @@
...
+
+.error {
+  color: red;
+}
+
```
Comment (Garrett-Iannuzzi) [inline src/css/base.scss]: Possibly use a variable here if all errors will be in red?

File: src/css/base.scss (hunk 1 above)
Comment (benjamin-firth) [reply src/css/base.scss]: That's a great idea, thanks!  Definitely better to tackle that now while I don't have too many css variables.

File: src/css/base.scss (hunk 2 above)
Comment (benjamin-firth) [reply src/css/base.scss]: Good call, I'll make that change for future errors!

**Comment by benjamin-firth** [comment]
@Garrett-Iannuzzi Thanks for the review, I forgot about the variables for things like error color so that's super helpful.  I definitely need to work on more focused PRs, so it's good to have the reminder.  Go ahead and approve it!

**Pull request merged**